README.md

```diff
@@ -88,6 +80,25 @@ Create a `appsettings.json` file in the AELFFaucet.Web directory of the project
   }
 }
 ```
+
+## Setup secrets.json file
+
+Run this following command in terminal of the AELFFaucet.Web directory to generate the secrets.json file with all ENV keys: 
+
+```base title="Terminal"
+dotnet user-secrets init
+dotnet user-secrets set "ApiConfig:BaseUrl" "http://3.25.10.185:8000"
+dotnet user-secrets set "ApiConfig:PrivateKey" "b192e307da4c4e1a00eeed442a4b5e8e0b7b2c6f838d0472627b846e5280c51c"
```

Comment:
Also please do not expose keys in README.md. You can have a <insert your private key here> instead of an exact key.